src/Evolution/Executables/RelativisticEuler/Valencia/EvolveValencia.hpp

```diff
@@ -122,7 +123,7 @@ struct EvolutionMetavars {
       is_analytic_data_v<initial_data> xor is_analytic_solution_v<initial_data>,
       "initial_data must be either an analytic_data or an analytic_solution");
 
- using eos_base = typename EquationsOfState::get_eos_base<
+  using eos_base = typename EquationsOfState::get_eos_base<
```

Comment:
remove indent?

Reply:
I believe this is an action of `clang-format` and actually corrects a missing whitespace